test/spec/zeebe/ZeebeVariableResolver.spec.js

```diff
@@ -1043,32 +1044,65 @@ describe('ZeebeVariableResolver', function() {
 
   describe('parsing', function() {
 
-    beforeEach(bootstrapModeler(longBrokenExpressionXML, {
-      container,
-      additionalModules: [
-        ZeebeVariableResolverModule
-      ],
-      moddleExtensions: {
-        zeebe: ZeebeModdle
-      }
-    }));
+    describe('long broken expression', function() {
 
+      beforeEach(bootstrapModeler(longBrokenExpressionXML, {
+        container,
+        additionalModules: [
+          ZeebeVariableResolverModule
+        ],
+        moddleExtensions: {
+          zeebe: ZeebeModdle
+        }
+      }));
 
-    it('should NOT error on a long broken expression', inject(async function(elementRegistry, variableResolver) {
 
-      // given
-      const task = elementRegistry.get('Task_1');
-      const bo = getBusinessObject(task);
-      const input = getInputOutput(bo).inputParameters[1];
+      it('should NOT error on a long broken expression', inject(async function(elementRegistry, variableResolver) {
 
-      // when
-      const variables = await variableResolver.getVariablesForElement(bo, input);
+        // given
+        const task = elementRegistry.get('Task_1');
+        const bo = getBusinessObject(task);
 
-      // then
-      expect(variables).to.variableEqual([
-        { name: 'target' }
-      ]);
-    }));
+        // when
+        const variables = await variableResolver.getVariablesForElement(bo);
+
+        // then
+        expect(variables).to.variableEqual([
+          { name: 'target' }
+        ]);
+      }));
+
+    });
+
+
+    describe('immediately broken expression', function() {
+
+      beforeEach(bootstrapModeler(immediatelyBrokenExpressionXML, {
+        container,
+        additionalModules: [
+          ZeebeVariableResolverModule
+        ],
+        moddleExtensions: {
+          zeebe: ZeebeModdle
+        }
+      }));
+
+
+      it('should NOT error on an immediate syntax error', inject(async function(elementRegistry, variableResolver) {
+
+        // given
+        const task = elementRegistry.get('Task_2');
```

Comment:
I'd try to give elements that I reference semantic names, i.e. "TASK_WITH_BROKEN_EXPRESSION" in this case.

Reply:
Will do